modules/tinymce/src/themes/silver/main/ts/ui/menus/contextmenu/SilverContextMenu.ts

```diff
@@ -119,22 +142,37 @@ export const isTriggeredByKeyboard = (editor: Editor, e: PointerEvent) =>
 const getSelectedElement = (editor: Editor, e: PointerEvent) =>
   isTriggeredByKeyboard(editor, e) ? editor.selection.getStart(true) : e.target as Element;
 
-const shouldUseNodeAnchor = (editor: Editor, e: PointerEvent) => {
+const getPointAnchorSpec = (editor: Editor, e: EditorEvent<TouchEvent>) => {
+  const pointAnchor = getPointAnchor(editor, e);
+  if (isTouch()) {
+    return {
+      bubble: Bubble.nu(0, bubbleSize, bubbleAlignments),
+      layouts,
+      overrides: {
+        maxWidthFunction: MaxWidth.expandable(),
+        maxHeightFunction: MaxHeight.expandable()
+      },
+      ...pointAnchor
```

Comment:
Just confirming: which object do we want to take precedence if `pointAnchor` has bubble/layouts/overrides fields? I would have thought (without much context, so I could be wrong here) that we'd want to put `...pointAnchor` at the start of this object, instead of the end.

Reply:
Yeah, this all is incorrect. Mobile and Desktop have different requirements, so the bubble, layouts and anchor definitions need to remain in those files, not here.

Reply:
Hmm I moved and added an `if (isTouch())` so the code doesn't change - I mean from what was doing previously. But I agree that all those mobile config (layouts, bubble, ...) should be in the MobileContextMenu. I'll move it back.